src/ai/backend/manager/scheduler/dispatcher.py

```diff
@@ -231,6 +231,11 @@ async def schedule(
         Session status transition: PENDING -> SCHEDULED
         """
         log.debug("schedule(): triggered")
+        scheduler_status: dict[str, Any] = {
```

Comment:
```suggestion
        scheduler_status: dict[str, str] = {
```
just minor suggestion!